src/DripsHub.sol

```diff
@@ -105,12 +105,11 @@ contract DripsHub is Managed, Drips, Splits {
     /// between being taken from the users' drips balances and being receivable by their receivers.
     /// High value makes receiving cheaper by making it process less cycles for a given time range.
     /// Must be higher than 1.
-    /// @param reserve_ The address of the ERC-20 reserve which the drips hub will work with
-    constructor(uint32 cycleSecs_, IReserve reserve_)
+    constructor(uint32 cycleSecs_)
         Drips(cycleSecs_, erc1967Slot("eip1967.drips.storage"))
         Splits(erc1967Slot("eip1967.splits.storage"))
     {
-        reserve = reserve_;
+        return;
```

Comment:
Is it a compiler warning otherwise?

Reply:
It triggers `solhint`'s empty block warning.